README.md

```diff
@@ -93,6 +93,9 @@ if err != nil {
 }
 ```
 
+`Publish` will block until either the publish is acknowledged or failed to
+deliver. Alternatively you can use `PublishAsync` which does not block.
```

Comment:
I think it would be great if you can add an example to publish asynchronously. It will be great if the example is visible.
for `publishMultipleAsync`, you can just add a note below it.

Reply:
> devs who want to make sure message was published without using normal blockingpublish method

This seems very application specific?

In the connector we're just using it to monitor the number of failed messages
```
channel.PublishAsync(..., func(err error) {
  if (err != nil) {
    metrics.FailedPublishes.Inc()
  }
})
```

Could add the equivalent of the `Publish` example to just print the failed publish but that seems a bit redundant?

Reply:
yeah, I think you add example 
channel.PublishAsync(..., func(err error) {
  if (err != nil) {
    fmt.print("publish failed with err", err);
  } else {
      fmt.print("publish success")
    }
})